README.md

```diff
@@ -36,6 +36,8 @@ jobs:
 * **GITHUB_TOKEN** ***(required)*** - Required for permission to tag the repo.
 * **DEFAULT_BUMP** *(optional)* - Which type of bump to use when none explicitly provided (default: `minor`).
 * **WITH_V** *(optional)* - Tag version with `v` character.
+* **RELEASE_BRANCH** *(optional)* - Name of the branch that will generate the release tags. Other branches and pull-requests generate versions postfixed with the commit hash and do not generate any tag.
+* **RELEASE_BRANCH** *(optional)* - Comma separated list of branches (bash reg exp accepted) that will generate the release tags. Other branches and pull-requests generate versions postfixed with the commit hash and do not generate any tag.
```

Comment:
`RELEASE_BRANCH ` => `RELEASE_BRANCHES`

Reply:
Would it make more sense to just have a single ENV VAR, `RELEASE_BRANCHES` which defaults to `master`, rather than the two?

Reply:
I corrected the type. Regarding the name of the ENV VARs, I had a doubt. I started by defining only `RELEASE_BRANCHES`. Then , when applying it only to one branch (which is the most common: having just a mainline `master` or `develop`), I found it awkward and confusing... _For just one branch, why is it plural? Should I put a separator? ..._
But I am not fully satisfied either by having the two ENV VARs...

Maybe with just examples, it would clarify enough. I changed the code accordingly